### DISCUSSION TITLE
Fitur quiz

### DISCUSSION THREAD

**Discussion by devnazir** [upkoding/upkoding#14] [Ideas]
Kayaknya kalau ditambahin fitur quiz lebih menarik deh bli @ekaputra07, Jadi poin tidak hanya didapatkan dari menyelesaikan proyek. tapi bisa juga dari quiz yang dikerjakan. quiz nya tentang sekikar bahasa pemrograman atau fundamental pemrograman. Cuma saran aja sihh. hehee

**Comment by ekaputra07** [comment]
@devnazir menarik, apakah quiz ini merupakan bagian dari proyek? bukan kali ya... jadi terpisah gitu?

Klo dipikir2 bener juga sih, klo ngerjain proyek kan perlu dedicated time ya hehe jadi ga semua orang punya waktu cukup. tapi klo cuma sekedar klik2 jawab pertanyaan (pilihan ganda) + nambah ilmu mungkin bisa bikin orang lebih betah kali ya? 🙂

Tapi ntar ya habis forum diskusi ya, saya udah start soalnya hehe dan saya pikir penting juga, jadi nanti saya pengen klo ada apa yg kurang jelas dll biar ujung2nya lari ke diskusi/tanya-jawab.

**Reply by ekaputra07** [reply]
abis itu baru kita kembangkan lagi fitur belajarnya :)

**Reply by devnazir** [reply]
Iya terpisah dan cuma sekedar klik2 jawab pertanyan (pilihan ganda). 

oke bli

**Reply by ekaputra07** [reply]
@devnazir bagian terberat sebenarnya bukan bikin fiturnya tapi nyari daftar quiznya yang banyak 😁
kecuali kita bisa ambil quiz dari tempat lain yg bisa dipakai, meski bahasa inggris tak apa2 kita bisa translate

**Reply by devnazir** [reply]
Ambil dari sini aja
https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes

Lengkap banget.. wkwk

**Reply by ekaputra07** [reply]
Astaga! itu mah harta karun 😁  siip siip disimpan dulu 🙏
